data/generate.js

```diff
@@ -52,14 +52,14 @@ module.exports = () => {
   for (let order = 0; order < 50; order++) {
     const account = data.accounts[getRandomInt(data.accounts.length)];
     const id = data.orders.length + 1;
-    data.orders.push({
+    const _order = {
```

Comment:
It looks like this nukes the orders from `data.orders`, as I'm not seeing it added to them anywhere later on..,